examples/next-js-cms-blog/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "@builder.io/react": "2.0.2",
+    "@builder.io/widgets": "1.2.23",
+    "classnames": "2.3.1",
+    "date-fns": "2.28.0",
+    "next": "latest",
```

Comment:
please use a specific version for `next` to make sure this continue to work as expected n the future